test/e2e/misc_test.go

```diff
@@ -88,6 +87,48 @@ func (suite *MiscTestSuite) TestValidateBuildImage() {
 	}
 }
 
+// This is a test of the findRoute function, not a product test
+func (suite *MiscTestSuite) TestFindRoute() {
+	cleanupOptions := &framework.CleanupOptions{TestContext: ctx, Timeout: timeout, RetryInterval: retryInterval}
+
+	jaegerInstanceName := "simplest"
+	jaegerInstance := getSimplestJaeger(jaegerInstanceName, namespace)
+	err := fw.Client.Create(context.Background(), jaegerInstance, cleanupOptions)
+	require.NoError(t, err, "Error deploying example Jaeger")
+	defer undeployJaegerInstance(jaegerInstance)
+
+	// Create a second namespace and deploy another instance named "simplest"
+	secondContext, err := prepare(t)
+	if err != nil {
+		if secondContext != nil {
+			secondContext.Cleanup()
+		}
+		require.FailNow(t, "Failed in prepare with: "+err.Error())
+	}
+	defer secondContext.Cleanup()
+
+	secondNamespace, err := secondContext.GetNamespace()
```

Comment:
Wondering if the second namespace should be long, so it causes the name to be truncated?